editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -4981,7 +4981,7 @@ void VisualShaderEditor::_preview_size_changed() {
 	preview_vbox->set_custom_minimum_size(preview_window->get_size());
 }
 
-static ShaderLanguage::DataType _get_global_shader_uniform_type(const StringName &p_variable) {
+ShaderLanguage::DataType _visual_shader_editor_get_global_shader_uniform_type(const StringName &p_variable) {
```

Comment:
I asked about this a few days ago:
![image](https://github.com/godotengine/godot/assets/4701338/3d635bd4-0c0c-4ce2-8408-8efc0da43a94)

Sounds to me like we should avoid non-static and non-namespaced functions. Maybe this should be moved to the class?

Reply:
Ah yes this is a small historical mistake. I did first try moving it to the class before (hence removing the static) but got a bunch of compilation errors due to two definitions of `ShaderLanguage`. And aborted and moved it back, and forgot to change back the static.

I'll have another go and see if I can coax it into working as part of the class. :+1: